commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Activity.kt

```diff
@@ -1105,6 +1106,7 @@ private fun BaseSimpleActivity.renameCasually(
                             val tempDestination = try {
                                 createTempFile(File(sourceFile.path)) ?: return@updateSDK30Uris
                             } catch (exception: Exception) {
+                                toast(R.string.unknown_error_occurred)
```

Comment:
lets show the actual exception here with showErrorToast, just to get more info

Reply:
I see you fixed it. Thanks